build.sbt

```diff
@@ -20,13 +20,8 @@ buildInfoPackage := "com.jakehschwartz.finatra.swagger"
 buildInfoKeys := Seq[BuildInfoKey](name, version, swaggerUIVersion)
 
 libraryDependencies ++= Seq(
-  "com.twitter" %% "finatra-http" % twitterReleaseVersion excludeAll(
-    "com.fasterxml.jackson.core",
-    "com.fasterxml.jackson.dataformat",
-    "com.fasterxml.jackson.datatype",
-    "com.fasterxml.jackson.module"
-  ),
-  "io.swagger" % "swagger-core" % "1.6.1",
+  "com.twitter" %% "finatra-http" % twitterReleaseVersion,
+  "io.swagger" % "swagger-core" % "1.6.2",
   "io.swagger" %% "swagger-scala-module" % "1.0.6",
   "com.fasterxml.jackson.core" % "jackson-core" % jacksonVersion,
```

Comment:
Can we remove any of these dependencies like in https://github.com/jakehschwartz/finatra-swagger/pull/54 ?

Reply:
Done